jetty-rewrite/src/main/config/etc/jetty-rewrite.xml

```diff
@@ -10,7 +10,7 @@
       <New class="org.eclipse.jetty.rewrite.handler.RewriteHandler">
         <Set name="rewriteRequestURI" property="jetty.rewrite.rewriteRequestURI" />
         <Set name="rewritePathInfo" property="jetty.rewrite.rewritePathInfo" />
-        <Set name="originalPathAttribute" property="jetty.rewrite.originalPathAttribute" />
+        <Set name="originalPathAttribute"><Property name="jetty.rewrite.originalPathAttribute" default="requestedPath"/></Set>
```

Comment:
Ditto.

Reply:
See above comment.

Reply:
ditto

Reply:
Both @sbordet and @gregw see above comment.